roles/openshift_master/files/openshift-master.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Why not just use the unit files from the source repo if building the packages from source?

Also, any reason not to package the binaries up into an RPM, create a local yum repo and use that for installing the custom packages instead of just copying the binaries over?